### PR TITLE
`show-names` - Extend support of new issue view

### PR DESCRIPTION
**Pull request by kovsu**
- part of https://github.com/refined-github/refined-github/issues/7856

## Test URL

https://github.com/eslint/eslint/issues/18391

## Screenshot

before:

image

after:

image

### CODE REVIEW

**Comment by fregante** [comment]
Is it missing only from the first comment?

**Comment by busches** [comment]
This is already working for me on issues, https://github.com/refined-github/refined-github/issues/8022, and this PR. Is this tied to being in the beta or something else?

**Comment by fregante** [comment]
@kovsu please **always** include the reference to the issue and at least one test URL

@busches it's #7856

**Comment by kovsu** [comment]
> Is it missing only from the first comment?

yeah

**Comment by fregante** [comment]
Events lack it too: https://github.com/eslint/eslint/issues/18391#event-1518943450

Screenshot 5

But it's fine, we don't have to add it to every single username.

**Review by fregante — APPROVED** [review]
Otherwise you can merge as is

File: source/github-helpers/selectors.ts [hunk 1]
```diff
@@ -190,23 +190,27 @@ export const botLinksPrSelectors = [
...
+const authorLinks = [
+	'.js-discussion a.author',
+	'.inline-comments a.author',
+	'h3 a[data-testid="issue-body-header-author"]',
+	'.react-issue-comment a[data-testid="avatar-link"]',
+];
+
+// `a` selector needed to skip commits by non-GitHub users
+// # targets mannequins #6504
+// `show_full_name` is for GHE: https://github.com/refined-github/refined-github/issues/7232#issuecomment-1910803157
```
Comment (fregante) [inline source/github-helpers/selectors.ts:203]: These can now be inlined and the identifier dropped. e.g.
```
	'[show_full_name="true"]', // For GHE: https://github.com/refined-github/refined-github/issues/7232#issuecomment-1910803157
```

**Pull request merged**